src/DotVVM.Compiler/DotVVM.Compiler.csproj

```diff
@@ -19,6 +19,9 @@ for some reason. What fun. -->
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <RollForward>LatestMajor</RollForward>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+    <RuntimeIdentifiers>win7-x86;win7-x64</RuntimeIdentifiers>
+  </PropertyGroup>
```

Comment:
Why is this needed? The Compiler binaries created upon the build of DotVVM.CommandLine are framework-dependent, not self-contained, so this setting shouldn't really do much, right?

Reply:
I think this was necessary for the preview version, but I am not sure whether this is still needed.